plugins/tekton/README.md

```diff
@@ -76,13 +76,13 @@ The Tekton plugin enables you to visualize the `PipelineRun` resources available
     backstage.io/kubernetes-namespace: <RESOURCE_NS>
   ```
 
-- The following annotation is added to the `catalog-info.yaml` file of entity to view the latest `PipelineRun` in the CI/CD tab of the application:
+- The following annotation is added to the `catalog-info.yaml` file of the entity to enable the Tekton related features in Backstage. The value of the annotation identifies the name of the application:
 
   ```yaml
   annotations:
     ...
 
-    janus-idp.io/tekton-enabled : 'true'
+    janus-idp.io/tekton : <APP_NAME>
```

Comment:
It feels like this needs more explanation of where the <APP_NAME> is coming from.

Reply:
updated this to BACKSTAGE_ENTITY_NAME